src/components/Articles.jsx

```diff
@@ -1,27 +1,37 @@
-import { useEffect } from 'react';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
 import { getArticles } from '../utils/api';
 
 const Articles = () => {
-  getArticles();
   const [articles, setArticles] = useState([]);
+  const [err, setErr] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const { topic } = useParams();
 
   useEffect(() => {
-    getArticles().then((articlesFromApi) => {
-      setArticles(articlesFromApi);
-    });
-  }, []);
+    getArticles(topic)
+      .then((articlesFromApi) => {
+        setArticles(articlesFromApi);
+        setIsLoading(false);
+        setErr(null);
+      })
+      .catch((err) => {
+        setErr('Topic not found ☹️');
+      });
+  }, [topic]);
 
+  if (err) return <p>{err}</p>;
+  if (isLoading) return <p>Loading...</p>;
   return (
     <ul className="articles">
       {articles.map((article) => {
         return (
           <li className="listItems" key={article.article_id}>
             <h2>{article.title}</h2>
-            <h3>Author: {article.author}</h3>
-            <h3>Topic: {article.topic}</h3>
-            <h4>Votes: {article.votes}</h4>
-            <h4>Comments: {article.comment_count}</h4>
+            <p>Author: {article.author}</p>
```

Comment:
Not MVP, but at some point, it might be nice to be able to extract this out into its own component and make it reusable if we wanted to :)